### PR TITLE
Fix: Subtitle in RadioModalSettingsTile was not changing after more than two opens.

### PR DESCRIPTION
**Pull request by Juggler98**
[This commit](https://github.com/GAM3RG33K/flutter_settings_screens/commit/ef8e48a7e5e754b2fc29da05a374de41df7a3f8c) solved [this](https://github.com/GAM3RG33K/flutter_settings_screens/issues/90) and [this](https://github.com/GAM3RG33K/flutter_settings_screens/issues/81), but there was still a little problem with not updating subtitle in RadioModalSettingsTile after more than two opens. Check the [video](https://www.youtube.com/watch?v=SjEEg07jpbs). 

But I do not know, if this line was important for some other stuff like handling memory leaks, so merge with caution.

### CODE REVIEW

**Comment by alberto-paiva** [comment]
I changed it to:

```dart
@override
  void dispose() {
    if (mounted == false) {
      _notifiers.remove(cacheKey);
    }
    super.dispose();
  }
```

and it works ok here.

**Comment by Juggler98** [comment]
> I changed it to:
> 
> ```dart
> @override
>   void dispose() {
>     if (mounted == false) {
>       _notifiers.remove(cacheKey);
>     }
>     super.dispose();
>   }
> ```
> 
> and it works ok here.

I updated my pull request with this code. It works fine. @GAM3RG33K could you please merge this little change, so the plugin is working properly for everyone? :)

File: lib/src/settings.dart [hunk 1]
```diff
@@ -213,8 +213,10 @@ class _ValueChangeObserverState<T> extends State<ValueChangeObserver<T>> {
 
   @override
   void dispose() {
+    if (mounted == false) {
```
Comment (GAM3RG33K) [inline lib/src/settings.dart:216]: This can be written as `if (!mounted)`

**Review by GAM3RG33K — CHANGES_REQUESTED** [review]
Other than the small change in the `if` condition, there's nothing to worry about. 
@Juggler98  Please make the update & push it, will merge the PR ASAP

File: lib/src/settings.dart (hunk 1 above)
Comment (Juggler98) [reply lib/src/settings.dart]: Yes, it should be if (!mounted). I just copied the code from the comment above. I'm sorry. It's solved now.

**Review by GAM3RG33K — APPROVED** [review]
LGTM

**Pull request merged**